watch.go

```diff
@@ -21,7 +23,23 @@ type Event struct {
 	// Path to the data that changed
 	Path string
 	// Data that changed
-	Data interface{}
+	Data    interface{}
+	payload string
+}
+
+func (e Event) Value(v interface{}) (path string, err error) {
```

Comment:
I apologize that it took so long for me to response (life got a little crazy). In this case I think it would be worth changing the return type to just be `error` and continue to let uses access the path via the property. In the future, if the api is changed, then we could potentially double back and change this.